corehq/apps/users/views/mobile/users.py

```diff
@@ -1183,7 +1183,8 @@ class FilteredUserDownload(BaseManageCommCareUserView):
 
     @method_decorator(require_can_edit_commcare_users)
     def get(self, request, domain, *args, **kwargs):
-        form = CommCareUserFilterForm(request.GET, domain=domain, couch_user=request.couch_user)
+        form = CommCareUserFilterForm(request.GET, domain=domain,
+                                    couch_user=request.couch_user, is_first_load=True)
```

Comment:
I tried using `empty_permitted` and it seems to do what is needed, I have not tested it fully, so i leave that to you to try out
```
    @method_decorator(require_can_edit_commcare_users)
    def get(self, request, domain, *args, **kwargs):
        form = CommCareUserFilterForm(request.GET, domain=domain, couch_user=request.couch_user)
        form.empty_permitted = True
```
pinging @orangejenny if she has suggestions on this.

Reply:
`form.empty_permitted = True` is working fine.